library/classes/Prescription.class.php

```diff
@@ -255,6 +258,7 @@ function set_unit($unit) {
 
     function set_id($id) {
         if (!empty($id) && is_numeric($id)) { $this->id = $id; }
+        $this->id = $id;		
```

Comment:
I took this out because it is no longer used. I will make sure the changes get over this time.